### ISSUE TITLE
#emptyTonelSubdirectory reports failure even when succeeds

### ISSUE THREAD

**Issue by eMaringolo** [GemTalk/SETT#2]
The `OSUnixSubproccess` call succeeds (it effectively deletes the directory and the printString of it is `"OSSUnixSubprocess(Command: /bin/rm; Pid: 13406; Status: normal termination with status 0)"`).

But the `exitStatus` variable contains `139706696204288`, which is a non-zero return, and hence is reported as a failure in the command.

Changing this:
```smalltalk
	exitStatus := process waitForExit.
	exitStatus = 0 ifFalse: [...]
```
By this:
```smalltalk
	process waitForExit.
	process exitStatusInterpreter isSuccess ifFalse: [...]
```

Fixes it.

This happened with some git commands as well.

**Comment by kurtkilpela** [comment]
See PR #3

**Comment by kurtkilpela** [comment]
Merged PR #3
